### PR TITLE
Added new pagespeed insights link after set up.

### PR DESCRIPTION
**Pull request by ShahAaron**
## Summary

Addresses issue #532 

## Checklist

- [x] My code is tested and passes existing unit tests.
- [x] My code has an appropriate set of unit tests which all pass.
- [x] My code is backward-compatible with WordPress 4.7 and PHP 5.6.
- [x] My code follows the [WordPress](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) coding standards.
- [x] My code has proper inline documentation.
- [x] I have signed the Contributor License Agreement (see <https://cla.developers.google.com/>).

### CODE REVIEW

File: assets/js/components/notifications/notification.js [hunk 1]
```diff
@@ -146,6 +146,8 @@ class Notification extends Component {
 			module,
 			moduleName,
 			pageIndex,
+			pageSpeedLink,
+			pageSpeedLabel,
```
Comment (felixarntz) [inline assets/js/components/notifications/notification.js:150]: These props should be less specifically named since the component is generic, for example `anchorLink`, `anchorLinkLabel`.

Comment (ShahAaron) [reply assets/js/components/notifications/notification.js]: Added as requested here and in `dashboard-setup-alerts.js`

File: package-lock.json [hunk 2]
```diff
@@ -11695,7 +11695,7 @@
         },
```
Comment (felixarntz) [inline package-lock.json:11695]: The changes in this file seem unrelated. Will remove.
